components/brave_welcome_ui/components/svg/browser-icons/yandex.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+const SvgComponent = (props: any) => (
+<svg viewBox="0 0 200 200" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@aguscruiz can you pls help to get correct svg? This seems a bit bigger than expected, thanks